my-react-app/src/components/Buttons/AddButton.tsx

```diff
@@ -0,0 +1,53 @@
+import {useEffect, useRef } from 'react'
+import clsx from 'clsx';
+
+type Props = {
+    ButtonColor:string;
```

Comment:
props and variables bitte immer mit kleinen buchstaben starten und componenten mit großen buchstaben